code/game/objects/items/weebstick.dm

```diff
@@ -0,0 +1,194 @@
+/obj/item/weebstick
```

Comment:
Please name it better

Reply:
done, everything renamed to "nanoforgedkatana". if that's too long, i can change it again